common-issues.md

```diff
@@ -87,7 +87,7 @@ memory_limit = 512M
 Set `memory_limit` on the fly as a temporary fix:
 
 ```bash
-$ php -d memory_limit=512M wp package install <package-name>
+$ php -d memory_limit=512M $(which wp) package install <package-name>
```

Comment:
I think the `$(which wp)` should be in double quotes just in case there's spaces in the path.